bundles/CoreBundle/config/routing.yaml

```diff
@@ -26,7 +26,7 @@ _pimcore_service_thumbnail:
 _fos_js_routing:
     resource: "@FOSJsRoutingBundle/Resources/config/routing/routing-sf4.xml"
 
-# auto loads routes from all bundles in config/pimcore/routing.yml
+# auto loads routes from all bundles in config/pimcore/routing.yaml
```

Comment:
Thank you for the PR! 
This is actually covered o nhttps://github.com/pimcore/pimcore/pull/13077, it's a known issue with 11.x as we all have this problem until that PR get merged.
@robertSt7 Can you add this change in the comments, i have noticed that is missing from your PR

Reply:
@kingjia90 so we can close this in favor of https://github.com/pimcore/pimcore/pull/13077?